website/content/en/docs/building-operators/golang/tutorial.md

```diff
@@ -329,10 +329,16 @@ The name and tag of the image (`IMG=<some-registry>/<project-name>:tag`) in both
 
 #### Deploy the operator
 
-For this example we will run the operator in the `default` namespace which can be specified for all resources in `config/default/kustomization.yaml`:
+Per default a new namespace is created with name `<project-name>-system`. For this example we will run the operator in the `memcached-operator` namespace which can be specified for all resources in `config/default/kustomization.yaml`:
```

Comment:
this feels like it needs to reworded... at a min:
```suggestion
Per the default, a new namespace is created with name `<project-name>-system`. For this example we will run the operator in the `memcached-operator` namespace which can be specified for all resources in `config/default/kustomization.yaml`:
```

Reply:
It seems that "By default," is a more common use.